docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -14,13 +14,86 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-8.6.1>>
 * <<release-notes-8.6.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.6.1 relnotes
+
+[[release-notes-8.6.1]]
+== {fleet} and {agent} 8.6.1
+
+Review important information about the {fleet} and {agent} 8.6.1 release.
+
+[discrete]
+[[known-issues-8.6.1]]
+=== Known issues
+
+[discrete]
+[[known-issue-issue-2066-8-6-1]]
+.Osquery live query results can take up to five minutes to show up in {kib}
+[%collapsible]
+====
+*Details* +
+A known issue in {agent} may prevent live query results from being available
+in the {kib} UI even though the results have been successfully sent to {es}. 
+For more information, refer to {agent-issue}2066[#2066].
+
+*Impact* +
+Be aware that the live query results shown in {kib} may be delayed by up to 5 minutes.
+====
+
+[[known-issue-2170]]
+.Adding a {fleet-server} integration to an agent results in panic if the agent was not bootstrapped with a {fleet-server}
+[%collapsible]
+====
+
+*Details*
+
+A panic occurs because the {agent} does not have a `fleet.server` in the `fleet.enc`
+configuration file. When this happens, the agent fails with a message like:
+
+[source,shell]
+----
+panic: runtime error: invalid memory address or nil pointer dereference
+[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x557b8eeafc1d]
+goroutine 86 [running]:
+github.com/elastic/elastic-agent/internal/pkg/agent/application.FleetServerComponentModifier.func1({0xc000652f00, 0xa, 0x10}, 0x557b8fa8eb92?)
+...
+----
+
+For more information, refer to {agent-issue}2170[#2170].
+
+*Impact* +
+
+To work around this problem, uninstall the {agent} and install it again with
+{fleet-server} enabled during the bootstrap process.
+====
+
+[discrete]
+[[bug-fixes-8.6.1]]
+=== Bug fixes
+
+{fleet}::
+* Fix missing policy Id in installation URL for cloud integrations {kibana-pull}149243[#149243]
+* Fix package installation APIs to install packages without a version {kibana-pull}149193[#149193]
+* Fix issue where the latest GA version could not be installed if there was a newer prerelease version in the registry
+{kibana-pull}149133[#149133] {kibana-pull}149104[#149104]
+
+{fleet-server}::
+* No bug fixes in this release
+
+{agent}::
+* Fix issue where {beats} started by {agent} may fail with an `output unit has no config` error {agent-pull}2138[#2138] {agent-issue}2086[#2086]
+* Fix broken deployments that use traffic filter by injecting custom headers {agent-pull}2158[#2158] {beats-issue}32993[#32993]
```

Comment:
I still don't completely like the structure of this description (it's grammatically vague), but I think it's OK?

Reply:
We can reword this to something like "Restore the ability to set custom HTTP headers at enrolment time. Fixes traffic filters in Integrations Server cloud deployments."